lib/slack/message_button.rb

```diff
@@ -0,0 +1,74 @@
+require 'securerandom'
+
+module Slack
+  class MessageButton
+    #
+    # This method post a message to multiple users with Message Button
+    #
+    # @option params [String] :callback_id
+    # @option params [String] :slack_id
+    # @option params [Array<Hash>] :message_buttons
+    #
+    def bulk_post(params = {})
```

Comment:
> Message Posting
> In general we allow applications that integrate with Slack to send no more than one message per second. We allow bursts over that limit for short periods. However, if your app continues to exceed the limit over a longer period of time it will be rate limited.

要点としては

* 一般的には１秒につき１メッセージしか送信できない
* 但し短期間であれば制限を超えることを許可するが、長時間実行する場合はレートリミットがかけられる

ref: [Rate Limits \| Slack](https://api.slack.com/docs/rate-limits)

とのことなので、一旦はそのまま送信しても良さそうですが、後日ジョブキューから１秒おきに送信する仕組みにしたほうが良さそうです。

１秒以内に送信できるか試してみます。